server/models/Message.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema({  
+    conversationId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'conversation',
+      required: true,
+    },
+    senderProfileId:{
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'profile',
+      required: true,
+    },
+    content:{type:String, required: true},
```

Comment:
Where exactly is this going to be located where we can grab from it efficiently?

Reply:
I stored the messages of the conversation separately. we could get all messages of the conversation by using the conversation Id.